server/websocket.go

```diff
@@ -397,6 +404,39 @@ func (p *Plugin) wsReader(us *session, handlerID string) {
 			return
 		case <-us.rtcCloseCh:
 			return
+		case <-sessionAuthTicker.C:
+			// Server versions prior to MM v9.5 won't have the session ID set so we
+			// cannot go ahead with this check.
+			// Should be removed as soon as we bump the minimum supported version.
+			if authSessionID == "" {
+				continue
+			}
+
+			if s, appErr := p.API.GetSession(authSessionID); appErr != nil || (s.ExpiresAt != 0 && time.Now().UnixMilli() >= s.ExpiresAt) {
```

Comment:
When is the session's `ExpiresAt` set to 0 that we need to take into account here?

Reply:
I just saw your comment above that _We should also support sessions with a zero ExpiresAt which means no expiration._

A bit irrelevant to the PR, but when is there no session expiration?

Reply:
Thanks for explaining!